lib/spacesuit/auth_middleware.ex

```diff
@@ -52,6 +52,8 @@ defmodule Spacesuit.AuthMiddleware do
     result =
         token
         |> Joken.token
+        # is it possible to verify against alternatives here?
+        # that would make our lives easier if we want to rotate the secret / switch the algorithm
```

Comment:
Is it possible to verify against alternatives here?
That would make our lives easier if we want to rotate the secret / switch the algorithm